packages/blade/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -805,19 +807,46 @@ WithMultipleDropdowns.args = {
   name: 'design-system',
 };
 
-export const DropdownPerformance = (): React.ReactElement => {
+export const ASectionsPerf = (): React.ReactElement => {
```

Comment:
Renamed this? What is AsectionsPerf? DrowndownPerformance seemed more contextual

Reply:
Renamed to internal components so it doesn't show up on consumer end.